.github/workflows/go.yml

```diff
@@ -82,6 +82,18 @@ jobs:
     - name: Test registry server with PostgreSQL
       run: go test ./server/registry -postgresql
 
+    - name: Verify that benchmarks run
+      env:
+        APG_REGISTRY_ADDRESS: localhost:8080 
+        APG_REGISTRY_AUDIENCES: http://localhost:8080
+        APG_REGISTRY_INSECURE: 1
+        PROJECTID: bench
+        ITERATIONS: 1
+      run: |
+        registry-server &
```

Comment:
This isn't necessary because a server is started on line 65. In fact, it exists with an error because the port is in use by that server, but this doesn't block or affect the outcome of the test. I would take it out but I think @theganyo is likely to remove the server start on line 65 soon.